src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -1356,6 +1356,12 @@
   <data name="ERR_MethGrpToNonDel" xml:space="preserve">
     <value>Cannot convert method group '{0}' to non-delegate type '{1}'. Did you intend to invoke the method?</value>
   </data>
+  <data name="WRN_MethGrpToNonDel" xml:space="preserve">
+    <value>Converting method group '{0}' to non-delegate type '{1}'. Did you intend to invoke the method?</value>
```

Comment:
>'{0}' [](http://example.com/codeflow?start=35&length=5)

I would remove the '{0}' placeholder completely.

Reply:
I went for consistency with `ERR_MethGrpToNonDel` above.